### PR TITLE
update documentation in dockstore.cwl for newer tested cwltool version

### PR DESCRIPTION
**Pull request by junjun-zhang**
… version

### CODE REVIEW

File: Dockstore.cwl [hunk 1]
```diff
@@ -99,7 +99,7 @@ doc: |
     - Docker (1.12.6): follow instructions to install Docker https://docs.docker.com/engine/installation
     - CWL tool
     ```
-    pip install cwltool==1.0.20170217172322
+    pip install cwltool==1.0.20180116213856
```
Comment (denis-yuen) [inline Dockstore.cwl:102]: I'm assuming this is the version you're testing with?

File: Dockstore.cwl [hunk 2]
```diff
@@ -135,6 +135,7 @@ doc: |
         "path":"bams/normal_sample.bam",
         "class":"File"
       },
+      "download_reference_files": "false",
```
Comment (denis-yuen) [inline Dockstore.cwl]: Can we get this change into https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/blob/develop/Dockstore_cwl.json as well?

File: Dockstore.cwl (hunk 1 above)
Comment (junjun-zhang) [reply Dockstore.cwl:102]: correct

File: Dockstore.cwl (hunk 2 above)
Comment (junjun-zhang) [reply Dockstore.cwl]: added

**Review by denis-yuen — APPROVED** [review]
Noticed a couple more test json

**Pull request merged**